com.getferdi.Ferdi.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.getferdi.Ferdi
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: ferdi
+separate-locales: false
+rename-desktop-file: ferdi.desktop
+rename-icon: ferdi
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures:ro
+  - --filesystem=xdg-videos:ro
+  - --filesystem=xdg-documents:ro
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+modules:
+  - name: ferdi-desktop
+    buildsystem: simple
+    build-commands:
+      - ar x ferdi_*.deb
+      - rm -f ferdi_*.deb
+      - tar xf data.tar.xz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/* opt/* /app
+      - chmod -R a-s,go+rX,go-w /app/Ferdi
+      - rm -r /app/share/icons/hicolor/1024x1024
+      - desktop-file-edit --set-key=Exec --set-value='ferdi --no-sandbox %U' /app/share/applications/ferdi.desktop
+      - install ferdi.sh /app/bin/ferdi
+      - install -Dm644 com.getferdi.Ferdi.appdata.xml /app/share/appdata/com.getferdi.Ferdi.appdata.xml
```

Comment:
New appdata file should use `metainfo.xml` as a suffix and installed under `metainfo` instead of `appdata`